pkg/cmd/release-payload-controller/release_creation_job_controller.go

```diff
@@ -171,9 +136,35 @@ func (c *ReleaseCreationJobController) sync(ctx context.Context, key string) err
 		return err
 	}
 
+	// If the status is already set, then don't do anything
+	if len(originalReleasePayload.Status.ReleaseCreationJobResult.Status) > 0 {
+		return nil
+	}
+
 	releasePayload := originalReleasePayload.DeepCopy()
 
-	klog.V(4).Infof("Syncing ReleasePayload: %s/%s", releasePayload.Namespace, releasePayload.Name)
+	klog.V(4).Infof("Syncing ReleaseCreationJobResult for ReleasePayload: %s/%s", releasePayload.Namespace, releasePayload.Name)
+	releasePayload.Status.ReleaseCreationJobResult = v1alpha1.ReleaseCreationJobResult{
+		Coordinates: v1alpha1.ReleaseCreationJobCoordinates{
+			Name:      originalReleasePayload.Name,
+			Namespace: c.jobsNamespace,
+		},
+		Status: v1alpha1.ReleaseCreationJobUnknown,
```

Comment:
is this being set in two spots?

Reply:
Yes, it was.  I have updated the logic so that the 2 controllers now only write to their own respective pieces:
`ReleaseCreationStatusController`
1. .status.releaseCreationJobResult.status
2. .status.releaseCreationJobResult.Message

`ReleaseCreationJobController`
1. .status.ReleaseCreationJobResult.ReleaseCreationJobCoordinates.Namespace
2. .status.ReleaseCreationJobResult.ReleaseCreationJobCoordinates.Name